lib/services/faceRecognition.js

```
@@ -0,0 +1,131 @@
+      function loadModels(_x) {
+        return _loadModels.apply(this, arguments);
+      }
+      return loadModels;
+    }()
+  }, {
+    key: "compareFaces",
+    value: function () {
+      var _compareFaces = _asyncToGenerator(/*#__PURE__*/_regeneratorRuntime().mark(function _callee2(image1Tensor, image2Tensor) {
+        var detection1, detection2, distance, similarity;
+        return _regeneratorRuntime().wrap(function _callee2$(_context2) {
+          while (1) switch (_context2.prev = _context2.next) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)